addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -54,6 +54,13 @@ options.registry.mailing_list_subscribe = options.Class.extend({
         });
         return def;
     },
+    /**
+     * @see this.selectClass for parameters
+     */
+    toggleShowThanks(previewMode, widgetValue, params) {
+        this.$target[0].querySelector('.js_subscribe_btn').classList.toggle('d-none');
+        this.$target[0].querySelector('.js_subscribed_btn').classList.toggle('d-none');
+    },
```

Comment:
Using the same class as the public widget (d-none) to toggle the visibility of those buttons, don't you have the same problem that was solved for the countdown snippet with https://github.com/odoo/odoo/commit/c37354d457f5b868673b4f974e401f4c635062d2 as mentioned earlier ?